LaunchAtLogin/copy-helper.sh

```diff
@@ -9,7 +9,13 @@ mkdir -p "$helper_dir"
 cp -rf "$origin_helper_path" "$helper_dir/"
 
 defaults write "$helper_path/Contents/Info" CFBundleIdentifier -string "$PRODUCT_BUNDLE_IDENTIFIER-LaunchAtLoginHelper"
-codesign --force --entitlements="$CODE_SIGN_ENTITLEMENTS" --options=runtime --sign="$EXPANDED_CODE_SIGN_IDENTITY_NAME" "$helper_path"
+
+if [[ -z ${CODE_SIGN_ENTITLEMENTS} ]];
```

Comment:
I would prefer reversing this and checking that it exists. I think that's clearer. Using `-n`.

Reply:
Makes sense, done that :)